### DISCUSSION TITLE
是否有可能支持文字+图片的混合粘贴

### DISCUSSION THREAD

**Discussion by interjz4** [tamlok/vnote#2571] [General]
有时候需要复制一篇文章，包含了文字+多图，当前只能文字复制完成后，一张一张复制图片
可否支持一次性完成复制粘贴？

**Comment by tamlok** [comment]
你可以直接复制富文本，包括图片和文字，然后在vnote里面右键菜单，选择多功能粘贴，VNote会解析图片。
